app/models/events/new_discussion.rb

```diff
@@ -1,7 +1,8 @@
 class Events::NewDiscussion < Event
   def self.publish!(discussion)
     event = create!(kind: 'new_discussion',
-                    eventable: discussion)
+                    eventable: discussion,
+                    sequence_id: 0)
```

Comment:
@robguthrie Can you think of anything dangerous about doing this?

The current bug is as follows:

We have an auto-sequencer set up on event models which are in a 'discussion_id' channel.
We don't add a sequence id if event.discussion is nil
The 'discussion_id' of a 'new_discussion' event is nil, because the discussion id is stored in `eventable_id`

So, the 'new_discussion' event is sequenced, and it's sequenced in the 'discussion_id == nil' channel. Meaning it's equal to the number of discussions created since the sequencer was put in (~40k at this point)

The logic on the client side says something like 'if the minimum loaded event for this discussion has sequence id > discussion.firstSequenceId, show loadPreviousActivity'. This returns true if there is one event, and that event has sequence id ~40,000.